pkg/cmd/pulumi/stack_export.go

```diff
@@ -90,13 +90,6 @@ func newStackExportCmd() *cobra.Command {
 			}
 
 			if showSecrets {
-				// Currently, the stack.DefaultSecretsProvider is cached so adding a call to getStackSecretsManager
```

Comment:
Since this is now going to error properly, we can remove this workaround we added in this specific usecase